seeds/_sector/dex/elk_finance_avalanche_c_base_trades_seed.csv

```diff
@@ -0,0 +1,2 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+avalanche_c,elk_finance,1,2024-06-14,0x29588b5d8ec3bee204f05abeb218864ab03d50ef89e72f6273317ab4daa677a6,27,0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7,0xeeeeeb57642040be42185f49c52f7e9b38f8eeee,46711000,9858382649191728144,2817240000000000000000
```

Comment:
Please recheck this seed record.

Reply:
I just fixed @Hosuke. Thank you for pointing out